### PR TITLE
Add solution to change backend in windows

### PR DESCRIPTION
**Pull request by BraveDrXuTF**
I discover the whole document dosen't mention how to change backend in windows, and I add the solution to it.

### CODE REVIEW

File: docs/user/installation.rst [hunk 1]
```diff
@@ -72,11 +72,15 @@ DeepXDE supports TensorFlow 1.x (``tensorflow.compat.v1`` in TensorFlow 2.x), Te
 
     - Or ``export DDE_BACKEND=BACKEND`` to set the global environment variable
 
-* Modify the ``config.json`` file under "~/.deepxde":
+* In Linux, modify the ``config.json`` file under "~/.deepxde":
```
Comment (lululxvi) [inline docs/user/installation.rst:75]: This works for both Linux and Windows.

File: docs/user/installation.rst [hunk 2]
```diff
@@ -72,11 +72,15 @@ DeepXDE supports TensorFlow 1.x (``tensorflow.compat.v1`` in TensorFlow 2.x), Te
...
 
     - The file has the content such as ``{"backend": "tensorflow.compat.v1"}``
     - You can also use ``python -m deepxde.backend.set_default_backend BACKEND`` to set the default backend
 
+* In Windows, you can add system variable "DDE_BACKEND" to set the default backend.
```
Comment (lululxvi) [inline docs/user/installation.rst:80]: This should be moved to line 59.

File: docs/user/installation.rst (hunk 1 above)
Comment (BraveDrXuTF) [reply docs/user/installation.rst]: Yeah, I just find the .deepxde in user directory of my Windows system.

File: docs/user/installation.rst [hunk 3]
```diff
@@ -71,11 +71,16 @@ DeepXDE supports TensorFlow 1.x (``tensorflow.compat.v1`` in TensorFlow 2.x), Te
     $ DDE_BACKEND=paddle python pde.py
 
     - Or ``export DDE_BACKEND=BACKEND`` to set the global environment variable
+    - In Windows, you can add system variable "DDE_BACKEND" to set the default backend
```
Comment (lululxvi) [inline docs/user/installation.rst:74]: - Or set the global environment variable ``DDE_BACKEND`` as ``BACKEND``. In Linux, this usually can be done via ``export DDE_BACKEND=BACKEND``, and in Windows, add the environment variable "DDE_BACKEND" in System Settings.

File: docs/user/installation.rst [hunk 4]
```diff
@@ -71,11 +71,16 @@ DeepXDE supports TensorFlow 1.x (``tensorflow.compat.v1`` in TensorFlow 2.x), Te
...
 
 * Modify the ``config.json`` file under "~/.deepxde":
 
     - The file has the content such as ``{"backend": "tensorflow.compat.v1"}``
     - You can also use ``python -m deepxde.backend.set_default_backend BACKEND`` to set the default backend
+    - In Windows, you can find ``config.json`` file under "C:/Users/Username/.deepxde" directory.
+
```
Comment (lululxvi) [inline docs/user/installation.rst:81]: Delete these extra empty lines.

Comment (BraveDrXuTF) [reply docs/user/installation.rst]: Done

File: docs/user/installation.rst (hunk 3 above)
Comment (BraveDrXuTF) [reply docs/user/installation.rst]: 刚刚才发现这个修改意见，已经并成一行了。

**Pull request merged**